substrait_consumer/tests/functional/relations/aggregate_relation_snapshots/IsthmusProducer/aggregate_with_group_by_cube_plan.json

```diff
@@ -0,0 +1,300 @@
+{
+  "extensionUris": [{
+    "extensionUriAnchor": 1,
+    "uri": "/functions_arithmetic_decimal.yaml"
+  }],
+  "extensions": [{
+    "extensionFunction": {
+      "extensionUriReference": 1,
+      "name": "sum:dec"
+    }
+  }],
+  "relations": [{
+    "root": {
+      "input": {
+        "sort": {
+          "common": {
+            "direct": {
+            }
+          },
+          "input": {
+            "project": {
+              "common": {
+                "emit": {
+                  "outputMapping": [3, 4, 5]
```

Comment:
This project is hiding whether or not the aggregate outputs an additional column.

Reply:
could you explain a little more? how is it doing that?

Reply:
Hmm, so the output fields were 0, 1, 2 and they are being reexposed here as three fields.   If there was an extra column (needed if there are two or more grouping sets) then the columns should be 4, 5, 6.  It's not clear if they're treating the extra column as the first remapped column.  I suppose the correctness tests and types could validate that.